src/transformers/models/whisper/tokenization_whisper.py

```diff
@@ -584,7 +584,7 @@ def decode(
                 Whether or not to output the offsets of the tokens. This should only be set if the model predicted
```

Comment:
If we're using the `model` + `processor` API, then should we add an argument `output_word_offsets` that controls whether the tokenizer returns the word-level timestamps for each token? This would just be for consistency with Wav2Vec2, where passing this argument controls whether we return word level timestamps:
https://github.com/huggingface/transformers/blob/ff4c0fc7d2e8df5fb5971f9d8dbd1108edabf65b/src/transformers/models/wav2vec2/tokenization_wav2vec2.py#L562

Think we should strive to maintain consistency between the library where possible, so `output_offsets` can toggle the utterance level timestamps, and `output_word_offsets` toggles the word level timestamps